aws/aws-how-to/instances/launch-ubuntu-desktop.rst

```diff
@@ -1,78 +1,105 @@
-Launch an Ubuntu desktop on EC2
-===============================
+Launch and connect to an Ubuntu Mantic desktop EC2 instance
+===========================================================
 
-To create an Ubuntu desktop environment on your EC2 VM, you can use TightVNC as your remote desktop server and Remmina as your local desktop client.
+Use the xrdp server and Remmina to connect to an AWS EC2 instance of Ubuntu Mantis.
```

Comment:
This start sounds a little too much like an instruction rather than an introduction. The whole how-to guide is basically a set of instructions, but it is usually nice to have an introductory statement before the actual steps begin. It could be something along the lines of what we are trying to achieve and an overview of how we are going to do that. Your statement is exactly that, but could you try to make it sound less like an instruction? Also you missed the 'Mantis' :)